coral-component-table/src/scripts/TableRow.js

```diff
@@ -455,7 +455,7 @@ const TableRow = Decorator(class extends BaseComponent(HTMLTableRowElement) {
   /** @private */
   _toggleOrderable(orderable) {
     if (orderable) {
-      this._setHandle('coral-table-roworder');
+      this._setHandle('coral-table-roworder', 0);
```

Comment:
@Pareesh are you certain one `setTimeout` "tick" is enough of a delay?

Reply:
Hi @majornista ,
Ideally, the mutation is a microtask and it should get executed before the next macrotask(setTimeout). I tested this and it was working fine, maybe in future if we encounter such scenario again, then I will increase the timeout.